README.md

```diff
@@ -19,81 +19,265 @@
 This brings benefits, but it also provides grounds for attackers to exploit
-unsuspecting ML users. This is why Google launched the [Secure AI Framework
-(SAIF)][saif] to establish industry standards for creating trustworthy and
-responsible AI applications. The first principle of SAIF is to
-
-> Expand strong security foundations to the AI ecosystem
+unsuspecting ML users.
 
 Building on the work with [Open Source Security Foundation][openssf], we are
-creating this repository to demonstrate how the ML supply chain can be
-strengthened in _the same way_ as the traditional software supply chain.
+creating this collection of projects to strengthen the ML supply chain in
+_the same way_ as the traditional software supply chain.
 
-This repository hosts a collection of utilities and examples related to the
-security of machine learning pipelines. The focus is on providing *verifiable*
-claims about the integrity and provenance of the resulting models, meaning users
-can check for themselves that these claims are true rather than having to just
-trust the model trainer.
+The focus is on providing *verifiable* claims about the integrity and provenance
+of the resulting models, meaning users can check for themselves that these
+claims are true rather than having to just trust the model trainer.
 
-## Projects
+## Model Signing
 
-Currently, there are two main projects in the repository: model signing (to
-prevent tampering of models after publication to ML model hubs) and
-[SLSA](https://slsa.dev/) (to prevent tampering of models during the build
-process).
+This project demonstrates how to protect the integrity of a model by signing it.
+We support generating signatures via [Sigstore](https://www.sigstore.dev/), a
+tool for making code signatures transparent without requiring management of
+cryptographic key material. But we also support traditional signing methods, so
+models can be signed with public keys or signing certificates.
 
-### Model Signing
+The signing part creates a
+[sigstore bundle](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto)
+protobuf that is stored as in JSON format. The bundle contains the verification
+material necessary to check the payload and a payload as a
+[DSSE envelope](https://github.com/sigstore/protobuf-specs/blob/main/protos/envelope.proto).
+Further the DSSE envelope contains an in-toto statment and the signature over
+that statement. The signature format and how the the signature is computed can
+be seen
+[here](https://github.com/secure-systems-lab/dsse/blob/v1.0.0/protocol.md).
 
-This project demonstrates how to protect the integrity of a model by signing it
-with [Sigstore](https://www.sigstore.dev/), a tool for making code signatures
-transparent without requiring management of cryptographic key material.
+Finally, the statement itself contains subjects which are a list of (file path,
+digest) pairs a predicate type set to `model_signing/v1/model`and a dictionary
```

Comment:
I thought the predicate type was `https://model_signing/signature/v1.0`

Reply:
Great catch, I forgot to update